checker/manual/external-tools.tex

```diff
@@ -302,17 +302,17 @@ \subsection{Explanation\label{ant-task-explanation}}
 \section{Maven\label{maven}}
```

Comment:
I believe that the incompatibility with JDK 7 is intentional -- that is, the instructions only work with JDK 8.  Renato, is that correct?

If so, then we cannot use these as the only Maven instructions, because the Checker Framework supports JDK 7.  However, we could use these as alternate instructions that could be used by people who are using JDK 8.  When the Checker Framework drops support for JDK 7, then these could become the only instructions.